src/init.cpp

```diff
@@ -1018,6 +1023,36 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         return InitError(_("No proxy server specified. Use -proxy=<ip> or -proxy=<ip:port>."));
     }
 
+#if defined(USE_SYSCALL_SANDBOX)
+    if (args.IsArgSet("-sandbox") && !args.IsArgNegated("-sandbox")) {
+        const std::string sandbox_arg{args.GetArg("-sandbox", "")};
+        bool log_syscall_violation_before_terminating{false};
+        if (sandbox_arg == "log-and-kill") {
```

Comment:
I'd personally prefer "log-and-abort" and "abort" as options instead of "log-and-kill" and "kill". I know it's not a translated message, but it's slightly friendlier anyhow.

Reply:
Good idea: "abort" sounds better than "kill". Fixed!